imports/plugins/core/orders/server/no-meteor/mutations/createOrder.js

```diff
@@ -458,7 +458,7 @@ export default async function createOrder(context, input) {
   OrderSchema.validate(order);
   await Orders.insertOne(order);
 
-  appEvents.emit("afterOrderCreate", order).catch((error) => {
+  appEvents.emit("afterOrderCreate", order, context).catch((error) => {
```

Comment:
You don't need to pass the context since it's already passed to the startup function, and that's what you're using in your inventory startup function anyway.